ext/net/ops_tls.rs

```diff
@@ -1055,7 +1055,11 @@ where
     .with_safe_defaults()
     .with_no_client_auth()
     .with_single_cert(cert_chain, key_der)
-    .expect("invalid key or certificate");
+    .map_err(|e| {
```

Comment:
This should not print, but should rather return an error approximately like this:

`.map_err(|e| custom_error("InvalidData", format!("Error creating TLS certificate: {:?}", e)))`

Reply:
Fixed in [this commit](https://github.com/denoland/deno/pull/20157/commits/6fbf801705295076bda8db89efd4a43d8c45834f)